frontend/packages/dev-console/src/actions/modify-application.ts

```diff
@@ -41,3 +41,19 @@ export const EditApplication = (model: K8sKind, obj: K8sResourceKind): KebabOpti
     },
   };
 };
+
+export const EditHealthCheck = (model: K8sKind, obj: K8sResourceKind): KebabOption => {
+  return {
+    label: 'Edit Health Checks',
+    href: `/k8s/ns/${obj.metadata.namespace}/${obj.kind || model.kind}/${
+      obj.metadata.name
+    }/containers/${obj.spec.template.spec.containers[0].name}/health-checks`,
```

Comment:
You cannot use kind here.
eg. my knative service details page:
`http://localhost:9000/k8s/ns/cvogt/serving.knative.dev~v1~Service/react-web-app`
my health checks page for the same knative service:
`http://localhost:9000/k8s/ns/cvogt/Service/react-web-app/containers/user-container/health-checks`

expected to be:
`http://localhost:9000/k8s/ns/cvogt/serving.knative.dev~v1~Service/react-web-app/containers/user-container/health-checks`

need to use `referenceFor(obj)`

Reply:
Fixed it.